### PR TITLE
Update jacoco version

### LINKED ISSUE #34: Update JaCoCo to 0.8.7
I've encountered an issue 
```
Caused by: java.lang.IllegalStateException: Unexpected SMAP line: *S KotlinDebug
```

and it is related to [this reported issue](https://github.com/jacoco/jacoco/issues/1155)  

and fortunately for me it is now fixed today in version 0.8.7! We just to updated JaCoCo!

[JaCoCo 0.8.7 has been released](https://twitter.com/_godin_/status/1389728203342757891)
[JaCoCo Changelogs](https://www.jacoco.org/jacoco/trunk/doc/changes.html)

### PR DESCRIPTION
**Pull request by raquezha**
Update: I though this will fix issue 34

### CODE REVIEW

**Comment by Rolf-Smit** [comment]
Please see my comment here https://github.com/NeoTech-Software/Android-Root-Coverage-Plugin/issues/34#issuecomment-833095809

TLDR: Your PR doesn't actually fix #34, since this plugin is not enforcing a specific Jacoco version. However it can't hurt to update the JaCoCo version that is used by the integration test for this plugin.
